cli/ssh_test.go

```diff
@@ -475,6 +475,10 @@ Expire-Date: 0
 		// real error from being printed.
 		t.Cleanup(cancel)
 
+		// Wait for the prompt or any output really to indicate the command has
+		// started and accepting input on stdin.
+		_ = pty.ReadRune(ctx)
```

Comment:
If the goal is to wait for output why not make a method `WaitForOutput` instead or something, which has similar behavior but doesn't advance the reader (e.g. using Peek instead of Read)